bindings/go/blob/filesystem/file.go

```diff
@@ -13,6 +14,12 @@ import (
 
 const DefaultFileIOBufferSize = 1 << 20 // 1 MiB
 
+var (
```

Comment:
This looks like reinventing the wheel of already existing FS errors:
https://pkg.go.dev/io/fs#pkg-variables

Is there a specific reason you chose to go for your own variables?
If you want to specify Error Types for the Package, then I suggest to alias existing errors instead.